src/__tests__/resolver.test.ts

```diff
@@ -707,22 +711,28 @@ describe('ethrResolver', () => {
             {
               id: `${did}#service-3`,
               type: 'HubService',
-              serviceEndpoint: [{uri: 'https://hubs.uport.me', transportType: 'http'}, {uri: 'libp2p.star/123', transportType: 'libp2p'}],
+              serviceEndpoint: [
+                { uri: 'https://hubs.uport.me', transportType: 'http' },
+                { uri: 'libp2p.star/123', transportType: 'libp2p' },
+              ],
             },
           ],
         })
 
         // undo side effects of this test
         await new EthrDidController(identity, registryContract).revokeAttribute(
           stringToBytes32('did/svc/HubService'),
-          JSON.stringify([{uri: 'https://hubs.uport.me', transportType: 'http'}, {uri: 'libp2p.star/123', transportType: 'libp2p'}]),
```

Comment:
Not sure why Prettier touched theses lines. Just executed the normal formatting rules on each commit.

Reply:
no worries, this is not a blocker :)